app/grandchallenge/notifications/views.py

```diff
@@ -119,14 +120,29 @@ class FollowList(
     permission_required = "view_follow"
     filter_class = FollowFilter
     paginate_by = 50
-
-    def get_queryset(self, *args, **kwargs):
-        return (
-            super()
-            .get_queryset()
-            .select_related("user", "content_type")
-            .order_by("content_type")
-        )
+    excludes = [
+        ContentType.objects.filter(
+            app_label="archives", model="archivepermissionrequest"
+        ).get(),
+        ContentType.objects.filter(
+            app_label="algorithms", model="algorithmpermissionrequest"
+        ).get(),
+        ContentType.objects.filter(
+            app_label="reader_studies", model="readerstudypermissionrequest",
+        ).get(),
+        ContentType.objects.filter(
+            app_label="participants", model="registrationrequest"
+        ).get(),
+        ContentType.objects.filter(
+            app_label="cases", model="rawimageuploadsession"
+        ).get(),
```

Comment:
This does several DB calls, you should be able to get the same information out using `Q` queries, something like 

```
ContentType.objects.filter(
  Q(app_label="archives", model="archivepermissionrequest")
  | Q(app_label="algorithms", model="algorithmpermissionrequest")
  | ...
)
```

I think you could even use that directly in the exclude statement on L141 by doing something like 

```
Follow.objects.filter(
  Q(content_type__app_label="archives", content_type__model="archivepermissionrequest")
  | Q(content_type__app_label="algorithms", content_type__model="algorithmpermissionrequest")
  | ...
)
```

Don't know if that would be efficient though, would need to profile it.